docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/vol/web/media
+    command: >
+      sh -c "
+      python manage.py wait_for_db &&
+      python manage.py makemigrations &&
+      python manage.py migrate &&
+      python manage.py runserver 0.0.0.0:8000"
+    depends_on:
+      - db
+
+  db:
+    image: postgres:17.2-alpine3.20
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
```

Comment:
Ensure that the `$PGDATA` environment variable is set correctly in the `.env` file or in the environment where Docker is running. This variable is used to specify the location of the database files within the container.